api/api_interface_impl.go

```diff
@@ -103,8 +103,10 @@ func (i *OpenAPIInterfaceImpl) DisableBlocking(ctx context.Context,
 		}
 	}
 
-	if request.Params.Groups != nil {
+	if request.Params.Groups != nil && len(*request.Params.Groups) > 0 {
 		groups = strings.Split(*request.Params.Groups, ",")
+	} else {
+		groups = []string{}
```

Comment:
I think we can skip the `else` since usually a `nil` and an empty slice behave the same (`len(x) == 0` for both).

Reply:
Oh I just noticed I didn't get rid of the nil check. I'll do that shortly

Reply:
I think we do need that `nil` check cause it's for the pointer to the `Groups` string. It's likely `nil` when the query arg is absent.